### PR TITLE
Boos 310/fix/role permissions

### PR DESCRIPTION
**Pull request by koutsilisk**
In this PR we fix:
- Role model permissions to be read and write bool fields
- Role schema serialize/desirialize permissions dict to bool fields and vice versa.

Fixed:
- API contract was broken, not the backend follows the contract.

### CODE REVIEW

**Review by Chichilele — APPROVED** [review]
Much better this way, isn't it? The next step will be probably a separate model for fine permissions to different areas of the app

File: backend/project/api/schemas/role.py [hunk 1]
```diff
@@ -29,3 +31,19 @@ def deserialize_group_field(self, data, many, **kwargs):
             except NoResultFound as e:
                 raise e  #  with the name of the group
         return data
+
+    @post_dump
+    def serialize_permissions(self, data, many, **kwargs):
+        if "read" in data and "write" in data:
```
[the remark this answers is not in the document]
Comment (koutsilisk) [reply backend/project/api/schemas/role.py:37]: should we allow that?

[the remark this answers is not in the document]
Comment (koutsilisk) [reply backend/project/api/schemas/role.py:37]: will update

[the remark this answers is not in the document]
Comment (koutsilisk) [reply backend/project/api/schemas/role.py:37]: btw realized this is dump not load :) , you have to have both read-write when dumping from obj to json

[the remark this answers is not in the document]
Comment (Chichilele) [reply backend/project/api/schemas/role.py:37]: haha true

**Pull request merged**